backend/Services/CsvReader.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Services;
+
+class CsvReader
+{
+    public function getCSVData(string $fullPath): array
+    {
+        $csvRows = [];
+        $csvFile = fopen($fullPath, "r");
```

Comment:
We can use here `try {} catch()` and throw exception to detect problems with file opening